### ISSUE TITLE
Visualization libraries

### ISSUE THREAD

**Issue by Nurkoo** [Matyyas/short_text_topic_modeling#1]
Hello!
Love your work, this implementation is really great. I'm trying to use it on my own data set and I have encountered a problem - you're importing libraries like visualisation or topic_allocation, but those are never described - would you mind sharing them? I love the output of these functions, but can't quite understand and rewrite that code by myself. I do especially care for your implementation of pyLDAvis on that dataset - I can't think of a way to create a topic - term matrix needed for that.

Thanks for this code, I love it!

**Comment by palex-91** [comment]
Hi,
I agree with Nurek, nice work! But unfurtunately I am facing the same problem with visualisation and topic_allocation. Your help would be very much appreciated!
Thanks!

**Comment by mamrou** [comment]
Hi, 
Thank you for your feed-back!

I built those functions inside my companies source code and can't share them publicly. However, you might find here some code and insights build such functions:
- for [topic_allocation](https://github.com/blmoistawinde/gsdmm/blob/master/README.md).
- regarding visualisation, I studied the input of pyLDAvis plot methods and preprocessed my data to fit it.

Let me know 🙂
